.buildkite/pipeline.yml

```diff
@@ -1,7 +1,7 @@
 # Nodes with values to reuse in the pipeline.
 common_params:
   plugins: &common_plugins
-  - &bash_cache automattic/bash-cache#2.0.0: ~
+    - automattic/a8c-ci-toolkit#2.14.0
```

Comment:
Technically speaking, latest version of the plugin is now `2.15.0` as Gio made a new release recently.

But the only new thing in `2.15.0` is improvements in the `annotate_build_failures` command, which isn't used in this repo's `.buildite/commands` anyway, so no big deal 🙃